pkgs/applications/radio/sdrpp/default.nix

```diff
@@ -1,45 +1,57 @@
 { stdenv, lib, fetchFromGitHub, cmake, pkg-config
-, libX11, glfw, glew, fftwFloat, volk, AppKit
+, libX11, glfw, glew, fftwFloat, volk, zstd, AppKit
 # Sources
 , airspy_source ? true, airspy
 , airspyhf_source ? true, airspyhf
-, bladerf_source ? false, libbladeRF
+, bladerf_source ? true, libbladeRF
 , file_source ? true
 , hackrf_source ? true, hackrf
-, limesdr_source ? false, limesuite
-, sddc_source ? false
-, rtl_sdr_source ? true, rtl-sdr, libusb1
+, limesdr_source ? true, limesuite
+, perseus_source ? false    # needs libperseus-sdr, not yet available in nixpks
+, plutosdr_source ? stdenv.isLinux, libiio, libad9361
+, rfspace_source ? true
+, rtl_sdr_source ? true, rtl-sdr-osmocom, libusb1  # osmocom better w/ rtlsdr v4
 , rtl_tcp_source ? true
 , sdrplay_source ? false, sdrplay
 , soapy_source ? true, soapysdr
 , spyserver_source ? true
-, plutosdr_source ? stdenv.isLinux, libiio, libad9361
+, usrp_source	? true, uhd, boost
+
 # Sinks
 , audio_sink ? true, rtaudio
-, portaudio_sink ? false, portaudio
 , network_sink ? true
+, portaudio_sink ? false, portaudio
+
 # Decoders
 , falcon9_decoder ? false
 , m17_decoder ? false, codec2
 , meteor_demodulator ? true
 , radio ? true
-, weather_sat_decoder ? true
+, weather_sat_decoder ? false  # is missing some dsp/pll.h
+
 # Misc
 , discord_presence ? true
 , frequency_manager ? true
 , recorder ? true
 , rigctl_server ? true
+, scanner ? true
 }:
 
 stdenv.mkDerivation rec {
   pname = "sdrpp";
-  version = "1.0.4";
+
+  # SDR++ uses a rolling release model.
+  # Choose a git hash from head and use the date from that commit as
+  # version qualifier
+  git_hash = "3e58d4ba31eb0e78e85afdd811d80a32249fce19";
+  git_date = "20240121";
+  version = "1.1.0-" + git_date;
```

Comment:
Why not `2024-01-21`? Do we really need `1.1.0` prefix?

Reply:
Done, changed to iso8601 date-format 2024-01-21.

I would _not_ like to drop the 1.1.0 version as
  * all [upstreaam packaging](https://github.com/AlexandreRouma/SDRPlusPlus/blob/master/make_debian_package.sh#L11) has the 1.1.0 prefix. The current version is assumed to be [1.1.0 with nightly builds](https://github.com/AlexandreRouma/SDRPlusPlus/releases/tag/nightly)
  * if we ever switch to a stable version 1.2.0, then this would be _smaller_ if we had a version named 2024-mm-dd. Having the current upstream-provided version as prefix makes sure things will work out correctly.

Reply:
my understanding of https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

is that version should be `1.1.0-unstable-2024-01-21`, keeping 1.1.0 prefix makes sense